### DISCUSSION TITLE
Null response from Wordpress REST API

### DISCUSSION THREAD

**Discussion by mateusz-wyka** [nuxt/nuxt#16118] [General]
Hi

I am working on headless wordpress and I am trying to fetch data using `useFetch`, `useLazyFetch`, `useAsyncData` or `useLazyAsyncData`. I get a null response almost every time. I rarely get data. Adding `await nextTick();` helps partly, but it doesn't make my problem 100% solved. 

I will add that when, for example, I use `useFetch` and give an external address to the API, for example: https://api.nuxtjs.dev/mountains, there are no problems.

Some code

```
const data = await useFetch(`${runtimeConfig.public.baseURL}${API_NAV_MENU}`);
```

The `${runtimeConfig.public.baseURL}${API_NAV_MENU}` returns `http://localhost/wp-json/menus/v1/menus/2`.

After `console.log(data);`
![obraz](https://user-images.githubusercontent.com/89961396/207011968-97c16a36-0c09-4b47-9c04-05d4dadd420a.png)

Any ideas?

**Comment by wobsoriano** [comment]
baseURL

https://nuxt.com/docs/api/composables/use-fetch/#params

**Reply by mateusz-wyka** [reply]
I have `baseURL` set.
As I wrote above: The `${runtimeConfig.public.baseURL}${API_NAV_MENU}` returns `http://localhost/wp-json/menus/v1/menus/2.`

**Reply by Amirali-Yavari** [reply]
I think your api have problem
